docs/recipes/Cancellation.md

```diff
@@ -1,6 +1,6 @@
 # Cancellation
 
-Cancelling some async side effects is a common requirement of Epics. While there are several ways of doing this depending on your requirements, the most common way is to have your application dispatch a cancellation action and listen for it inside your Epic.
+Canceling some async side effects is a common requirement of Epics. While there are several ways of doing this, depending on your requirements, the most common way is to have your application dispatch a cancellation action and listen for it inside your Epic.
```

Comment:
If we change cancelling to canceling we need to do the spelling style change everywhere. There is at least one missed (I haven’t checked for others) the heading “Cancellation”

Reply:
Ah the file itself too, maybe other places. Otherwise feel free to undo these changes as another option.

Reply:
Also remember to change any links too. It’s unfortunate that changing the file will cause 404s for SEO. Wonder what could be done easily.

Reply:
I'll revert those changes.

Reply:
Actually, I'll take a better look at the docs and see if I can improve them. There seems to be a load of old Rxjs references and some outdated answers.